lib/src/properties_parser.dart

```diff
@@ -168,7 +183,7 @@ class Line {
   String get keyString => String.fromCharCodes(_key).trim();
 
   /// Get the value as a String.
-  String get valueString => utf8.decode(_value).trim();
+  String get valueString => utf8.decode(_value, allowMalformed: true).trim();
```

Comment:
Not sure about `allowMalformed`. It means we're just hiding the input error. Why do you think we should do that?

Reply:
In some cases user may want non UTF-8 characters to be parsed. I think we should allow user to choose to allow not UTF-8 characters or not.

Reply:
This library doesn't support that at the moment. It would need additional tests so I'd rather not make the change since it's not related to the point of the PR (null-safety support)

Reply:
Okay I will update the PR.